pyproject.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 python = "^3.9"
 fastapi = "^0.65.2"
 uvicorn = {extras = ["standard"], version="^0.18.2"}
-gunicorn = ">=20.0.4"
+gunicorn = "^22.0.0"
```

Comment:
I'd like to keep all our deps >= until we hit a version that doesn't work, then we create a ticket, pin, and try to fix soon

We're not consistent with this, but in newer services, we've tried to keep every >=